src/ansys/mapdl/core/mapdl.py

```diff
@@ -3204,9 +3204,29 @@ def run(
         if "\n" in command or "\r" in command:
             raise ValueError("Use ``input_strings`` for multi-line commands")
 
+        # Checking kwargs
```

Comment:
```suggestion
        # Check kwargs
```
Google developer documentation style guide recommends using the simple verb tense (without the -ing ending).